client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -96,6 +96,44 @@ Undocumented.prototype.jetpackModulesDeactivate = function( siteId, moduleSlug,
 	return this.wpcom.req.post( { path: '/sites/' + siteId + '/jetpack/modules/' + moduleSlug }, {}, { active: false }, fn );
 };
 
+/*
+ * Retrieve the settings of a Jetpack module with moduleSlug on the site with id siteId
+ *
+ * @param {int} [siteId]
+ * @param {string} [moduleSlug]
+ * @param {Function} fn
+ * @api public
+ */
+Undocumented.prototype.fetchJetpackModuleSettings = function( siteId, moduleSlug, fn ) {
+	//@TODO: implement and test this endpoint, it's currently not working
+	return this.wpcom.req.get( {
+		path: '/jetpack-blogs/' + siteId + '/rest-api/',
+		body: {
+			path: '/module/' + moduleSlug
```

Comment:
Ah, thanks for clarifying. 

Well, the reason is that it simply does not work this way. When I try using `query` instead of `body` in a GET request, the query args are not added at all. Haven't dug in-depth to see why this is happening, but that was the main reason behind my usage of `body` instead.